raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -41,54 +41,6 @@ def get_list_of_block_numbers(item):
     return list()
 
 
-def test_call_inexisting_address(deploy_client, blockchain_backend):
```

Comment:
why was where these tests removed?

Reply:
They either test methods that have been removed (`eth_call`...) or they do test behavior that's been replaced by `web3` library calls.

Reply:
Here some assumptions are tested, so it still makes sense to check that web3 follows them. Not sure about the `eth_call` though.

Reply:
`eth_call` shouldn't be used, that's what proxies are for (they should provide interface to interact with contract and use `web3.eth.contract.functions.call()` internally)

(that's the case for first three tests, the other with `estimate_gas` _might_ be worth keeping on a second thought)

Reply:
These were **not** tests for the `eth_call` function, please don't remove tests you don't understand. This was added to make sure our assumptions on the rest api are sane, and it caught a bug for when the `revert` opcode was introduced and these assumptions changed.

Reply:
@hackaugusto I've reverted the tests and updated them to use web3 calls; please take a look at it and decide if they are worth keeping.